dev/com.ibm.ws.org.apache.santuario.xmlsec.2.2.0/resources/org/apache/xml/security/resource/xmlsecurity_en.nlsprops

```diff
@@ -149,4 +184,43 @@ xml.WrongContent = Cannot find {0} in {1}
 xml.WrongElement = Cannot create a {0} from a {1} element
 xpath.funcHere.documentsDiffer = The XPath is not in the same document as the context node
 xpath.funcHere.noXPathContext = Try to evaluate an XPath which uses the here() function but XPath is not inside an ds:XPath Element. XPath was : {0}
-xpath.new.message.test = Test new message need to delete after test
+secureProcessing.MaximumAllowedTransformsPerReference = {0} transforms are contained in the Reference, maximum {1} are allowed. You can raise the maximum via the \"MaximumAllowedTransformsPerReference\" property in the configuration.
```

Comment:
{0} transforms are contained in the Reference. The maximum that is allowed is {1}. To increase this limit, set the \"MaximumAllowedTransformsPerReference\" property in the configuration.